package.json

```diff
@@ -33,7 +33,10 @@
         "matrix-org"
     ],
     "main": "./src/index.ts",
-    "browser": "./src/browser-index.ts",
+    "browser": {
```

Comment:
This field is managed by release.sh so that would need updating to not revert your fix

Reply:
gordon bennett. Why are we still using JSON as a config file format in 2023?

Reply:
thanks for the heads-up, anyway. My more recent attempt doesn't change this, so it's moot.